### DISCUSSION TITLE
Are Greek and Hebrew parsing available?

### DISCUSSION THREAD

**Discussion by cojolo1025** [ezra-bible-app/ezra-bible-app#1040] [Q&A] [answered]
I am a newbie with Ezra Bible and love it. I have discovered most of the features that I want are available in a very intuitive presentation. The only thing I haven't figured out how to do is how to display the simple Greek (and Hebrew) parsing. I know a few of the Greek texts that I have installed in Ezra Bible have them available. 

Is Greek parsing available? For me (a newbie), how do you do it?

Sincerely,

Joel Collier

**Comment by tobias-klein** [comment, marked as answer]
Hi @cojolo1025,

Thanks for your feedback!
If you mean the listing of grammatical details for individual words, that's currently not implemented. 
The only original language feature currently implemented is the Strong's based dictionary function.
